.github/workflows/website.yml

```
@@ -0,0 +1,77 @@
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.PERSONAL_GITHUB_TOKEN }}
+        repository: ThreeTen/threeten.github.io
+        path: target/threeten.github.io
+        ref: "main"
+
+    - name: Update website
+      run: |
+        cd target/threeten.github.io
+        git status
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)